### ISSUE TITLE
How to install ttx?

### ISSUE THREAD

**Issue by ygoe** [fonttools/fonttools#3010]
I somehow installed ttx.exe on Windows for Python 3.9 a year ago. Now I wanted to revisit my old project but can't figure out how to get the tools together. I found this repository that should provide the ttx command, and installed it again with `pip install fonttools`. That succeeded, but there is no `ttx` command available. I have found a file ttx.exe in my directory C:\Program Files\Python39\Scripts, but there is none in the current one C:\Program Files\Python311\Scripts which would also be in %PATH%.

So what do I have to do to get the `ttx` command on Windows? The documentation doesn't seem to cover this. Is it another Python package? Or a separate pip syntax? I can't get this working.

**Comment by jenskutilek** [comment]
Did you use the pip version matching your desired Python version (`pip3.11.exe`)?

Python 3.11 is usually installed in a subdirectory of your user directory, `AppData/Local/Programs/Python/Python311`, and `ttx.exe` should be in a subfolder `Scripts` there, which also needs to be in your PATH.

**Comment by ygoe** [comment]
Yes, it also suggested me to upgrade pip which I did afterwards. Python is installed in C:\Program Files for me.

I added that user profile directory to my user %PATH% variable but it wouldn't find the fonttools package (ModuleNotFoundError). Very strange.

After some more searching, I tried running pip from a command prompt with admin privileges. I first had to uninstall the package and then install it again. This time it landed in the Program Files directory and that works now. (I also had to upgrade pip *again* in that environment.) Maybe this needs to be installed as admin to make it work? I'm not very familiar with Python modules and how they're located.
